README.md

```diff
@@ -35,12 +35,23 @@ This module is optimally designed for use within a larger SIMP ecosystem, but it
 
 ### What useradd affects
 
-This module configures:
-  * `/etc/libuser.conf`
-  * `/etc/login.defs`
+This module can configure:
   * `/etc/default/nss`
   * `/etc/default/useradd`
+  * `/etc/group`
+  * `/etc/group-`
+  * `/etc/gshadow`
+  * `/etc/gshadow-`
+  * `/etc/libuser.conf`
+  * `/etc/login.defs`
+  * `/etc/passwd`
+  * `/etc/passwd-`
   * `/etc/profile.d/`
+  * `/etc/securetty`
+  * `/etc/security/opasswd`
+  * `/etc/shadow`
+  * `/etc/shadow-`
```

Comment:
Why does this module manage [backup files](http://man7.org/linux/man-pages/man5/shadow.5.html#FILES) like `group-`, `gshadow-`, `passwd-`, and `shadow-`?